packages/@glimmer/global-context/index.ts

```diff
@@ -14,6 +14,13 @@
  */
 
 import { DEBUG } from '@glimmer/env';
+import { assign } from '@glimmer/util';
+
+//////////
+
+export const FEATURES = {
```

Comment:
I just placed it near the top of the file, let me know if I should put it somewhere else.